structuredevent-model/src/main/java/com/sequenceiq/cloudbreak/structuredevent/event/rest/RestRequestDetails.java

```diff
@@ -19,7 +19,7 @@
 
     private Map<String, String> cookies;
 
-    @JsonSerialize(using = AnonymizingBase64Serializer.class)
+    @JsonSerialize(using = Base64Serializer.class)
```

Comment:
Rest request can have sensitive data in it

Reply:
@bbihari is taking a look a the performance impact of AnonymizingBase64Serializer